services/spar/spar.integration.yaml

```diff
@@ -14,6 +14,10 @@ saml:
       surname: Girlfriend
       email: email:president@evil.corp
 
+spInfo:
+  meta_uri:  https://app.wire.com/sso/metadata         # corresponds to 'APIMeta'
```

Comment:
What's the rationale behind mixing camelcase with underscore here?

Reply:
lazyness + ignorance of wire coding policy.  :-)

will fix.